frontend/pages/course-evaluation/[id].js

```diff
@@ -0,0 +1,22 @@
+import { useRouter } from 'next/router';
+import Card from '@mui/material/Card';
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+
+
+const Post = () => {
+  const router = useRouter();
+  const { pid } = router.query;
```

Comment:
I thought this should be the same name as the name of the file?

`[id].tsx` ... So the name of the variable should be `id`?

Notice in https://nextjs.org/docs/routing/dynamic-routes ... They use "pid" as the name of the file hence it's the name of the query parameter

Reply:
Changed it : )